ci/unit_tests.bat

```diff
@@ -1,6 +1,47 @@
 @echo off
+setlocal enabledelayedexpansion
 
-setlocal
+if not exist %WORKSPACE% (
```

Comment:
Is there a current working in batch (e.g. pwd)  that could be used instead of a the Jenkins environment variable ? (It is a bit kinder to Windows developers )

Reply:
yeah, once we know this work we can certainly improve it to not make it Jenkins specific somehow... or have some sensible fallback defaults.